### DISCUSSION TITLE
설치 완료후 localhost:8090 접속불가현상

### DISCUSSION THREAD

**Discussion by TeusEE** [lablup/contribution-academy-2021#86] [Q&A] [answered]
Describe the problem
설치 완료 후 backend.ai-webserver를 킨 이후에  web-ui를 사용하기 위해서 접근 시 localhost:8090로 접근하면 접근거부로 더이상 진행이 불가능
확실하지는 않지만, WSL2 환경에서 무선랜을 사용할 경우 WSL2호스팅 윈도우10에서 localhost접근이 안되는 현상으로 추정됩니다.
WSL2 호스팅 -> 윈도우10 클라이언트 접속 불가
(port 충돌때문인가 하여 18090으로 변경하여도 역시 접근거부 발생)
![image](https://user-images.githubusercontent.com/79349894/130965149-09ab3a71-0bd9-44ba-82e5-bc7da1a84ddc.png)

WSL2 내부에서 접근할 경우 정상적인 접근 환인
![image](https://user-images.githubusercontent.com/79349894/130972881-6ee3d89d-74bc-4248-89b7-8f91b50fbd46.png)

윈도우10을 호스팅서버로 react를 테스트했을 경우, 보시는것처럼 정상적으로 localhost에 접근이 가능했습니다
![image](https://user-images.githubusercontent.com/79349894/130967336-00d5c0b8-89b9-4070-9f95-18d666d6c86f.png)

To Reproduce
WSL2상에서 무선랜으로 backend.ai-webserver가동 -> localhost:8090으로 접근

Expected behavior
정상적인 webui loading

Screenshots
![image](https://user-images.githubusercontent.com/79349894/130965218-c69f9243-96bf-4bae-8a3b-439331aef585.png)
![image](https://user-images.githubusercontent.com/79349894/130965265-e6f15742-348e-4a1e-b10b-db0b0183f49a.png)

Environment Description
OS : WSL2 Ubuntu 20.04 LTS
Current shell : Bash
무선랜 환경 사용 중

**Comment by lizable** [comment, marked as answer]
@TeusEE Please refer this posting: https://www.sysnet.pe.kr/2/0/12347
